.github/actions/build-frontend/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
     - name: Use Node.js ${{ matrix.node-version }} with yarn
       uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
       with:
-        node-version: "16.19.1"
+        node-version: "18.14.2"
```

Comment:
I wonder if we can just use [`node-version-file`](https://github.com/actions/setup-node/pull/338) so we can just maintain the version in .nvmrc instead of having to keep this updated as well.

Reply:
@JosiahSiegel feasible?

Reply:
If devops approves, I feel this would be the right way to go.

Reply:
Also, according to the docs (see: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file) it can read the engines.node property in package.json so the .nvmrc file may not be needed.

Reply:
Oh, fantastic!  Anything automatic like this would be the way to go in my opinion 🙏